tests/project_helpers.py

```diff
@@ -275,6 +297,7 @@ def from_toml(cls, name: str, data: TomlData) -> "BaseExperiment":
 
     def get_venv_dir(self, cache: pytest.Cache) -> Path:
         """Get this venv's dir and create it if necessary."""
+        print(f"EXPERIMENT REQUIREMENTS: {self.requirements}")
```

Comment:
Intentional, or temporary debug to be removed?

Reply:
Removed.